lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -91,3 +91,10 @@
     margin-right: 20px;
   }
 }
+.fixed {
+  position: fixed;
+  top: 0;
+  z-index: 9999;
```

Comment:
Please use a UI-depth Sass extend here rather than an arbitrary value - https://github.com/edx/edx-platform/blob/master/lms/static/sass/base/_mixins.scss#L98.